types/yargs/index.d.ts

```diff
@@ -7,6 +7,7 @@
 //                 Jimi (Dimitris) Charalampidis <https://github.com/JimiC>
 //                 Teddy Cross <https://github.com/tkazec>
 //                 Steffen Viken Valvåg <https://github.com/steffenvv>
+//                 Tim De Pauw <https://github.com/timdp>
```

Comment:
Are you signing up to review future changes to the file?  If not I wouldn't bother adding your name to the list.